src/cephadm/cephadm.py

```diff
@@ -2792,7 +2800,7 @@ def command_bootstrap(ctx):
         if not os.path.isfile(ctx.custom_prometheus_alerts):
             raise Error(f'No custom prometheus alerts file found at {ctx.custom_prometheus_alerts}')
 
-    (user_conf, _) = get_config_and_keyring(ctx)
+    _, _ = get_config_and_keyring(ctx)
```

Comment:
If we're dropping the user of `user_conf` we should probably just remove this line. I don't think `get_config_and_keyring` has any side effects that would require us to still run it.

Reply:
There is a cephadm unit test that checks for the case where the `--config` argument is passed to the `bootstrap` subcommand but the file does not exist. This check happens in `get_config_and_keyring`. If we remove it, the unit test will fail.

Reply:
Ah, I see. I think I'd prefer we adjust the test somehow rather than having a redundant line, but I won't press for it.

Reply:
Haha, I think so too.